src/content/subgames.h

```diff
@@ -34,6 +34,7 @@ struct SubgameSpec
 	int release;
 	std::string path;
 	std::string gamemods_path;
+	std::string gameid_alias;
```

Comment:
This should be a `std::vector<std::string>` of normalized strings ideally. This way we could avoid duplicating the parsing on the Lua side.